proto/prysm/v1alpha1/beacon_block.proto

```diff
@@ -64,10 +67,13 @@ message GenericBeaconBlock {
 
         // Representing a post-eip4844 fork beacon block.
         BeaconBlockWithBlobKZGs eip4844 = 5;
+
+        // Representing a post-eip4844 fork beacon block.
+        BlindedBeaconBlockWithBlobKZGs blinded_eip4844 = 6;
     }
 
     // Representing an eip4844 fork blobs sidecar.
-    BlobsSidecar sidecar = 6;
+    BlobsSidecar sidecar = 100;
```

Comment:
same here. It's safe because the `v1alpha1` protos are used internally between the validator and the beacon node. They aren't part of the Beacon API spec.